packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -63,11 +63,18 @@ export interface AxisConfig {
   axisWidth?: number; // pixels ideally auto?
 }
 
+export interface SeriesConfig extends Record<string, any> {
```

Comment:
why Record?

Reply:
https://github.com/grafana/grafana/pull/29575/files#diff-9dc503bbea5a0385b2a5675eb77afd7ebbc71cd4c294d7cbcd744e77c457ee81R15

We will get a type error about not being able to iterate over it if we don't have it.

Reply:
but this removes a lot of type checking from the type. Might be better to just typecast it to Record in the place you want to iterate over it